backend/src/chats/message/message.spec.ts

```diff
@@ -34,27 +56,60 @@ describe('MessageController', () => {
 				ConfigModule.forRoot({ isGlobal: true }),
 				TypeOrmModule.forRootAsync({ useClass: TypeOrmConfigService }),
 				TypeOrmModule.forFeature([Message, Chat]),
+				UserModule,
+				AuthModule,
+				SharedModule,
+				AnimalModule,
+				ChatModule,
+				MessageModule,
+				UserModule,
+				RoleModule,
+				GameModule,
+				UserChatModule,
+				GameInvitesModule,
+				MatchmakingRequestModule,
 			],
 			controllers: [MessageController, ChatController],
 			providers: [MessageService, ChatService],
 		}).compile();
 
 		messageController = testingModule.get<MessageController>(MessageController);
 		chatController = testingModule.get<ChatController>(ChatController);
+
 		messageService = testingModule.get<MessageService>(MessageService);
 		chatService = testingModule.get<ChatService>(ChatService);
 
+		userController = testingModule.get<UserController>(UserController);
+		userService = testingModule.get<UserService>(UserService);
+
```

Comment:
For these things, they are kinda optional. As long as you don't need them, you don't really have to define them.
It's also not causing any problems. But in this case, you are not using the chatService and the userController. Again, not a breaking problem or anything, just to know :)